source/_data/glossary.yml

```diff
@@ -282,6 +282,19 @@
     other integrations.
   link: /docs/configuration/platform_options/
 
+- term: Reload
+  definition: >-
+    Updates Home Assistant configuration files. Changes are normally
```

Comment:
The first sentence is actually incorrect and misleading.

It states: "Updates Home Assistant configuration files", but it doesn't update or touch those at all...

Reply:
When you put it like that, you're right it doesn't make changes to the configuration files. I'll raise a new PR.